src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -104,8 +105,22 @@ export default class MultiscaleImageLayer extends CompositeLayer {
         loaderSelection,
         signal
       });
+      const isInterleavedAndRGB = isInterleaved && isRgb;
       if (tile) {
-        tile.data = noWebGl2 ? to32BitFloat(tile.data) : tile.data;
+        tile.data =
+          // eslint-disable-next-line no-nested-ternary
+          noWebGl2 && !isInterleavedAndRGB
+            ? to32BitFloat(tile.data)
+            : isInterleavedAndRGB
+            ? tile.data[0]
+            : tile.data;
+      }
+      if (tile.data.length === tile.width * tile.height * 3) {
+        return {
+          ...tile,
+          format: GL.RGB,
+          dataFormat: GL.RGB
+        };
       }
       return tile;
```

Comment:
Might have similar logic to what I suggested in the `ImageLayer`

Reply:
```javascript
        if (isInterleaved && isRgb) {
          tile.data = tile.data[0];
          if (tile.data.length === tile.width * tile.height * 3) {
            tile.format = GL.RGB;
            tile.dataFormat = GL.RGB;
          }
          return tile;
        }

        if (noWebGl2) {
          tile.data = to32BitFloat(tile.data);
        }
        return tile;
```

Reply:
Done.  Great suggestion!